internal/transport/transport.go

```diff
@@ -355,6 +365,27 @@ func webWorker(ctx context.Context, workerID int, jobs <-chan *webJob) {
 			job.logger.Fatal(err)
 		}
 
+		if !json.Valid(bytes) {
+			if job.flattenedRequest.clobColumn == "" {
+				job.repoJobs <- nil
+				msg := fmt.Sprintf("response body for %s was invalid JSON, "+
+					"discarding data since no 'clobColumn' was defined in the configuration file",
+					job.fetchConfig.URL)
+				logInfo := tools.LogFormatter{Msg: msg}
+				job.logger.Warnf(logInfo.String())
+
+				continue
+			}
+
+			data := make(map[string]string)
+			data[job.flattenedRequest.clobColumn] = string(bytes)
```

Comment:
```suggestion
			data[job.flattenedRequest.clobColumn] = string(bytes)
			
```
There is one last annoying linting error concerning cuddling. Just need to add an empty line here.